### PR TITLE
chore: Updating latest requirements to fix broken CI

### PR DESCRIPTION
**Pull request by franciscojavierarceo**
# What this PR does / why we need it:
Updating latest requirements to fix broken CI

# Which issue(s) this PR fixes:
Master CI is broken

# Misc

### CODE REVIEW

**Comment by franciscojavierarceo** [comment]
@tokoko can you reproduce this issue locally? I'm not able to reproduce this linter error.

**Comment by tokoko** [comment]
no, I'm usually running lowest python version locally though (3.9), maybe it is 3.11-specific.

**Comment by franciscojavierarceo** [comment]
I tried it with 3.11 and wasn't able to reproduce...

@dmartinol any chance you can check this out?

File: sdk/python/requirements/py3.10-ci-requirements.txt [hunk 1]
```diff
@@ -458,7 +460,7 @@ multidict==6.1.0
     # via
     #   aiohttp
     #   yarl
-mypy==1.11.2
+mypy==1.13.0
```
Comment (dmartinol) [inline sdk/python/requirements/py3.10-ci-requirements.txt:463]: revert this one.
I think the issue is due to this change in [1.11.2](https://github.com/python/mypy/blob/master/CHANGELOG.md#inferring-unions-for-conditional-expressions)

**Review by emgeee — APPROVED** [review]

**Review by lokeshrangineni — APPROVED** [review]

**Pull request merged**